tests/helm/template/federated-overrides.yml

```diff
@@ -0,0 +1,10 @@
+cloudAccountId: "1234567890"
+clusterName: "my-cluster"
+region: "us-east-1"
+apiKey: "049a3c8b-7259-4ad3-931a-73596c647cf8"
```

Comment:
I know this is a fake api key, but maybe in the future it might be a good idea to called it something like `fake-key` just to be super explicit